staker/commands.go

```diff
@@ -33,6 +33,8 @@ type stakingRequestCmd struct {
 type stakeExpansionReqFields struct {
 	prevActiveStkTxHash           *chainhash.Hash
 	prevActiveStkStakingOutputIdx uint32
+	prevStakingAmount             btcutil.Amount
```

Comment:
wouldn't it be easier to ask for an `additionalAmount`, might affect the way we handle the new staking amount